src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/Defaults.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Microsoft.Extensions.AI.Evaluation.Reporting;
+
+/// <summary>
+/// A static class that contains default values for various reporting artifacts.
+/// </summary>
+public static class Defaults
+{
+    /// <summary>
+    /// The default execution name that should be used if one was not specified as part of the
+    /// <see cref="ReportingConfiguration"/>.
+    /// </summary>
+    public const string DefaultExecutionName = "Default";
+
+    /// <summary>
+    /// The default iteration name that should be used if one was not specified when creating a
+    /// <see cref="ScenarioRun"/> via
+    /// <see cref="ReportingConfiguration.CreateScenarioRunAsync(string, string, IEnumerable{string}?, CancellationToken)"/>.
+    /// </summary>
+    public const string DefaultIterationName = "1";
+
+    /// <summary>
+    /// A <see cref="TimeSpan"/> that specifies the default amount of time that cached AI responses should survive
+    /// in the <see cref="IResponseCacheProvider"/>'s cache before they are considered expired and evicted.
+    /// </summary>
+    public static readonly TimeSpan DefaultTimeToLiveForCacheEntries = TimeSpan.FromDays(14);
```

Comment:
This should be a property rather than a field.

Do the consts above need to be consts, in order to be used in only places consts are valid, or could they also be made properties?

Properties give you flexibility to change them in the future, both their values and the implementation. Consts don't provide any hooks for code to run and get baked into the call sites.

Reply:
I've converted the `static readonly` field above to a property.

The `const`s above need to be preserved since they are used exclusively as default values of method parameters.